draft-hendrickson-privacypass-public-metadata.md

```diff
@@ -212,18 +204,17 @@ Upon receipt, the Client handles the response and, if successful, processes the
 content as follows:
 
 ~~~
-authenticator = Finalize(pkI, nonce, metadata, blind_sig, blind_inv)
+authenticator = Finalize(pkI, nonce, extensions, blind_sig, blind_inv)
 ~~~
 
-Where `Finalize` function is defined in {{Section 4.4 of !PBLINDRSA}}.
+Where `Finalize` function is defined in {{Section 4.4 of !PBRSA}}.
 
-If this succeeds, the Client then constructs a Token as described in [AUTHSCHEME] as
+If this succeeds, the Client then constructs a Token as described in {{AUTHSCHEME}} as
 follows:
 
 ~~~
 struct {
   uint16_t token_type = 0xDA7A; /* Type Partially Blind RSA (2048-bit) */
-  opaque metadata<1..2^16-1>;
```

Comment:
Do we need the "extensions" in the final Token struct as it is public metadata available at all points?

Reply:
As discussed in Slack, it's carried alongside the Token.

Reply:
SGTM